apps/ui/lib/lens/common/ViewFooter/ViewFooter.tsx

```diff
@@ -14,7 +14,9 @@ import * as notifications from '../../../services/notifications';
 import { RelationshipContract } from 'autumndb';
 
 const Footer = styled(Flex)`
-	border-top: 1px solid #eee;
+	position: absolute;
```

Comment:
This causes problems for lenses that don't infinite scroll, like the table lens. We may need to conditionally fix the button based on the lens being rendered 🤔

Reply:
After some though I think the best approach will be to give controls to lenses to declaratively not render the footer, and adjust the other lenses to compensate for a floating button.